flights/screens/SignUp.js

```diff
@@ -1,14 +1,51 @@
 import { Button, TextInput, View, Text } from "react-native";
+import { useEffect, useState } from "react";
 import { Formik } from "formik";
 import { styles } from "../styles/styles";
 import { AntDesign } from "@expo/vector-icons";
-import React, { useState } from "react";
 import Checkbox from "expo-checkbox";
+// import { getAuth, createUserWithEmailAndPassword } from "firebase/auth"
+import { collection, addDoc, getDocs } from "firebase/firestore"; 
+import { db } from "../firebaseConfig";
+// import firebase from 'firebase/app';
+
+import { app } from "../firebaseConfig";
+
+const collectionRef = collection(db,'users')
+
 
 export default function SignUp() {
+  const [users, setUsers] = useState([])
+  useEffect(()=>{
+    const getUsers = async ()=>{
+      console.log("üsers!!!")
+      await getDocs(collectionRef).then((users)=>{
+        let usersData = users.docs.map((doc)=> ({...doc.data(), id:doc.id}))
+        setUsers(usersData)
+      }).catch((err)=>console.log(err))
```

Comment:
Do not leave debug code on GitHub.

Instead, you can make use of Alert.alert()